pkg/system/system.go

```diff
@@ -167,8 +170,19 @@ func (sc *Controller) registerRouter() {
 	sc.router.HandleFunc(endpointDaemons, sc.describeDaemons()).Methods(http.MethodGet)
 	sc.router.HandleFunc(endpointDaemonsUpgrade, sc.upgradeDaemons()).Methods(http.MethodPut)
 	sc.router.HandleFunc(endpointDaemonRecords, sc.getDaemonRecords()).Methods(http.MethodGet)
+	sc.router.HandleFunc(endpointPrefetch, sc.getPrefetchFiles()).Methods(http.MethodGet)
```

Comment:
Http Get method should be used when client wants to query nydus-snapshotter's states. I am suggesting using Put method here